src/actions/smartWalletActions.js

```diff
@@ -1503,8 +1503,8 @@ export const estimateSmartWalletDeploymentAction = () => {
 export const checkArchanovaSessionIfNeededAction = () => {
   return async (dispatch: Dispatch, getState: GetState) => {
     // skip check if no archanova account
-    const archanovaAccountExists = !!findFirstArchanovaAccount(accountsSelector(getState()));
-    if (!archanovaAccountExists) return;
+    // const archanovaAccountExists = !!findFirstArchanovaAccount(accountsSelector(getState()));
+    // if (!archanovaAccountExists) return;
```

Comment:
Just remember to remove any unused code - if you're unsure if it'll be needed later, leave a comment above explaining why you commented it out (in this case I think you can just remove it though)

Reply:
I've removed the commented code